src/web/src/components/SearchBar.tsx

```diff
@@ -33,107 +35,114 @@ const useStyles = makeStyles((theme: Theme) =>
         padding: theme.spacing(0, 0, 0, 2),
       },
     },
-    header: {
-      padding: 0,
-      marginBottom: theme.spacing(2),
-      backgroundColor: theme.palette.background.default,
-    },
-    h1: {
-      display: 'block',
-      transition: 'all linear 350ms',
-      fontWeight: 600,
-      color: theme.palette.text.secondary,
-      [theme.breakpoints.between('xs', 'sm')]: {
-        fontSize: '3rem',
-      },
-      [theme.breakpoints.between('md', 'lg')]: {
-        fontSize: '4rem',
-      },
-      [theme.breakpoints.up('xl')]: {
-        fontSize: '5rem',
+    input: {
+      fontSize: '1.8rem',
+      width: '100%',
+      boxSizing: 'border-box',
+      height: '56px',
+      outline: 'none',
+      border: 'solid 1px transparent',
+      borderRadius: '50px',
+      paddingLeft: '55px',
+      background: '#EEEEEE',
+      position: 'absolute',
+      transition: 'border .2s',
+      '&:hover': {
+        border: 'solid 1px #999999',
       },
     },
     iconButton: {
-      backgroundColor: '#999999',
+      backgroundColor: '#EEEEEE',
+      color: '#999999',
       '&:hover': {
-        backgroundColor: theme.palette.secondary.dark,
+        backgroundColor: '#EEEEEE',
       },
       '& * > .MuiSvgIcon-root': {
         fontSize: '2rem',
-        color: theme.palette.primary.contrastText,
       },
       margin: 0,
       position: 'absolute',
-      right: '10px',
+      left: '10px',
       top: '6px',
     },
-    selectControl: {
-      '& > *': {
-        width: 'auto',
-        transform: 'translateY(2px)',
-        fontSize: '1.5rem',
-        textTransform: 'capitalize',
-        color: theme.palette.primary.main,
-        paddingLeft: '2rem',
-        [theme.breakpoints.down('xs')]: {
-          paddingLeft: '1rem',
-          width: '10rem',
-          transform: 'translateY(15px)',
-        },
+    clearIcon: {
+      color: '#999999',
+      '&:hover': {
+        backgroundColor: '#EEEEEE',
       },
-    },
-    selectItem: {
-      fontSize: '1.4rem',
-      textTransform: 'capitalize',
-      color: theme.palette.primary.main,
+      '& * > .MuiSvgIcon-root': {
+        fontSize: '3rem',
+      },
+      margin: 0,
+      position: 'absolute',
+      right: '5px',
+      top: '1px',
     },
   })
 );
 
 const SearchBar = () => {
   const classes = useStyles();
+  const [keyword, setKeyword] = useState('');
```

Comment:
keyword and setKeyword can be replaced by text and onTextChange from useSearchValue();

Remove this